backend/src/modules/doubles-tournament/core/application/command/StartTournamentCommandHandler.ts

```diff
@@ -0,0 +1,25 @@
+import { CommandHandler } from '../../../../../shared/core/application/command/CommandHandler';
+import { StartTournament } from './StartTournament';
+import { DomainEventPublisher } from '../../../../../shared/core/application/event/DomainEventBus';
+import { CurrentTimeProvider } from '../../../../../shared/core/CurrentTimeProvider';
+import { CommandResult } from '../../../../../shared/core/application/command/CommandResult';
+import { DoublesTournamentRepository } from '../DoublesTournamentRepository';
+import { tournamentWasStarted } from '../../domain/DoublesTournament';
+
+export class StartTournamentCommandHandler implements CommandHandler<StartTournament> {
+  constructor(
+    private readonly eventPublisher: DomainEventPublisher,
+    private readonly currentTimeProvider: CurrentTimeProvider,
+    private readonly repository: DoublesTournamentRepository,
+  ) {}
+
+  async execute(command: StartTournament): Promise<CommandResult> {
+    const tournament = await this.repository.findByTournamentId(command.tournamentId);
+
+    const { state, events } = tournamentWasStarted(tournament, this.currentTimeProvider());
```

Comment:
Co Ty @Szambelan . Cały projekt robiłeś dobrze, a teraz zacząłeś odwalać :P ? 
A może robiłeś ślepo podążając za Mistrzem bez zrozumienia :D 

Ta metoda, to nie może być w czasie przeszłym - dobrze było - ty karze zrobić startTournament (to jest jak command), a dopiero jej wynik to jest event tournamentWasStarted i może być też więcej eventów ewentualnie. Nie wiesz też czy się powiedzie czy nie.

Reply:
kurde nie tą nazwę zmieniłem -.-
faktycznie ... nie wiem o co biega z tym co ja tu tworzą

Reply:
dobra - poprawione ... masakra ze mną :-P